### PR TITLE
fix for deployment: added the sk param support

### PR DESCRIPTION
**Pull request by ig0r**

### CODE REVIEW

**Review by DieMyst — APPROVED** [review]

File: quickstart/5-oracle-service/scripts/deploy.sh [hunk 1]
```diff
@@ -1,8 +1,30 @@
 #!/usr/bin/env bash -o errexit -o nounset -o pipefail
 
+function is_installed {
+      if command -v "$1" >/dev/null 2>&1; then
+         echo "${1} is installed"
```
Comment (folex) [inline quickstart/5-oracle-service/scripts/deploy.sh:5]: why do you need to echo it every time?

File: quickstart/5-oracle-service/scripts/deploy.sh [hunk 2]
```diff
@@ -1,8 +1,30 @@
...
+         exit 1
+      fi
+}
+
+is_installed jq
+is_installed aqua
+
+echo "generating keys..."
+
+SK=$(aqua key create | jq -r ."secretKey")
```
Comment (folex) [inline quickstart/5-oracle-service/scripts/deploy.sh:18]:
```suggestion
SK=$(aqua key create | jq -r ".secretKey")
```

File: quickstart/5-oracle-service/scripts/deploy.sh (hunk 1 above)
Comment (ig0r) [reply quickstart/5-oracle-service/scripts/deploy.sh]: yep, removed it

File: quickstart/5-oracle-service/scripts/deploy.sh (hunk 2 above)
Comment (ig0r) [reply quickstart/5-oracle-service/scripts/deploy.sh]: fixed, thanks!

**Review by folex — APPROVED** [review]

**Pull request merged**